source/administration/object-management/data-compression.rst

```diff
@@ -0,0 +1,191 @@
+.. _minio-data-compression:
+
+================
+Data Compression
+================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+
+Overview
+--------
+
+MinIO Server supports compressing objects to reduce disk usage.
+Objects are compressed before they are written to disk and uncompressed when read, making the compresson process transparent to client applications and services.
+
+Depending on the type of data, compression may also increase overall throughput.
+Write throughput for compressible data is generally 500MB per second or greater per available CPU.
+Decompression is typically at least 1 GB per second.
```

Comment:
How specific should we be about performance here? These numbers are from the readme.

Reply:
@klauspost to confirm, but I think we should avoid numbers as it sets expectations that depend on the user having a CPU that is not a potato.

We could keep it general and say "Deployments following MinIO's recommended production configurations typically see..." and that should do it?

Reply:
I think there are enough qualifiers already (generally/typically).

Reply:
I rewrote this paragraph a bit, including using Ravind's text.